core/ir/opnd_api.h

```diff
@@ -1307,6 +1307,268 @@ enum {
 #endif
 };
 
+/* We need a seprate DR_REG_V enum from DR_REG_ so we can start counting virtual registers
+ * from lower values.  Otherwise, the DR_REG_ enum values won't fit in the 1 byte operand
+ * size of #DR_ISA_REGDEPS encoding.  Note that DR_REG_V skip the value of DR_REG_INVALID
```

Comment:
s/skip the value of/skips both values of/

Reply:
Done.